docs/pages/management/security/general-recommendations.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: General Security Recommendations
+description: Provides a simplified checklist of security measures to consider to protect enrolled resources and Teleport clusters.
+---
+
+If you host the Teleport cluster on your own network, review the following checklist for actions you can take to make 
+your infrastructure and Teleport cluster more secure and less vulnerable to different attack vectors. You can alleviate
+many of these security concerns by using Teleport Team or Teleport Cloud Enterprise.
+
+- Use specific role combinations to assign users the least privilege.
+- Consider separating people from roles to ensure checks and balance.
```

Comment:
Is the alternative assigning one role per user? I guess I'm not sure what this means. I would clarify "checks and balance" as well.